<launch>        
    <!--
       Start an apriltag node using the default settings
       to connect to an OpenNI device (e.g. Kinect).
    -->
    <arg name="base_kinect_name" default="kinect2_roof"/>
    <node pkg="apriltags_kinect_fusion" type="apriltags_kinect_fusion" name="apriltags_kinect_fusion" output="screen">
        <param name="default_tag_size" value="0.037" />

        <!--<param name="~viewer" value="false" />-->
        <!--<param name="~publish_detections_image" value="false" />-->

        <!--<param name="~display_type" value="CUBE" />-->
        <!--<param name="~marker_thickness" value="0.02" />-->

        <param name="image_topic"       type="string" value="$(arg base_kinect_name)/hd/image_color_rect"/>
        <param name="point_cloud_topic" type="string" value="$(arg base_kinect_name)/hd/points"/>
        <param name="camera_info_topic" type="string" value="$(arg base_kinect_name)/hd/camera_info"/>
    </node>
</launch>
